### PR TITLE
Add grid

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary

Add grid

## Screenshots

gridd

## Trello Card

https://trello.com/c/ULPh3XYv/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

**Review by gabrielizalo — APPROVED** [review]

File: maquetado/scss/description.scss [hunk 1]
```diff
@@ -2,34 +2,67 @@
...
 
 .description-book-cover {
   background-color: $white;
   box-shadow: 5px 5px 10px 0 $black-05;
-  display: flex;
-  height: 415px;
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  grid-template-rows: 40px 40px 1fr;
+  grid-template-areas:
```
Comment (gabrielizalo) [inline maquetado/scss/description.scss]: Please move `grid-template-areas` before `grid-template-columns`.

File: maquetado/scss/description.scss [hunk 2]
```diff
@@ -197,3 +177,57 @@
...
 }
+
+@media only screen and (max-width: 1024px) {
+  .description-book-cover {
+    background-color: $white;
+    box-shadow: 5px 5px 10px 0 $black-05;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr 1px 1fr;
+    grid-template-areas:
```
Comment (gabrielizalo) [inline maquetado/scss/description.scss]: Move `grid-template-areas` before `grid-template-columns`.

File: maquetado/scss/description.scss [hunk 3]
```diff
@@ -197,3 +177,57 @@
...
+  .description-content-book-content {
+    grid-area: content;
+    justify-self: center;
+    margin-left: 0;
+    position: relative;
+    &::before {
+      background-color: $green;
+      content: "";
+      height: 5px;
+      left: 4px;
```
Comment (gabrielizalo) [inline maquetado/scss/description.scss]: Can `4px` be `5px`?

**Review by gabrielizalo — CHANGES_REQUESTED** [review]

File: maquetado/scss/description.scss [hunk 4]
```diff
@@ -2,34 +2,67 @@
...
+  grid-area: genre;
+  justify-self: center;
+  line-height: 40px;
+}
+
+.description-book-detail {
+  display: flex;
+  margin-bottom: 14px;
+  margin-left: 10px;
+  margin-top: 20px;
```
Comment (Dorian30) [inline maquetado/scss/description.scss]: Use the shorthand 

```suggestion
  margin: 20px 0 14px 10px;
```

File: maquetado/scss/description.scss [hunk 5]
```diff
@@ -41,30 +74,12 @@
...
-    width: 100%;
-  }
-
-  .description-image-container {
-    align-self: center;
-  }
-}
-
-.description-book-content-title {
+.description-content-book-content {
```
Comment (Dorian30) [inline maquetado/scss/description.scss]: Content-book-content ? :eyes:

**Review by Dorian30 — CHANGES_REQUESTED** [review]

File: maquetado/detailOfBooks.html [hunk 6]
```diff
@@ -19,11 +19,9 @@
           alt="Book Image"
         />
       </div>
-      <div class="description-book-content-title">
-        <div class="description-book-detail description-line-separator">
-          <h2 class="description-book-title">Titulo del Libro</h2>
-          <p class="description-book-genre">(género)</p>
-        </div>
+      <h2 class="description-book-title">Titulo del Libro</h2>
```
Comment (gabrielizalo) [inline maquetado/detailOfBooks.html:22]: Please add tilde to `Titulo` in this way `Título`.

**Review by gabrielizalo — APPROVED** [review]

**Review by Dorian30 — APPROVED** [review]

**Pull request merged**